cabal.project

```diff
@@ -48,6 +48,8 @@ allow-newer:
            -- The following two dependencies are needed by plutus.
            , eventful-sql-common:persistent
            , eventful-sql-common:persistent-template
+           -- The following is needed for BRAL benchmarking
+           ,cardano-binary:recursion-schemes
```

Comment:
boo, looks like this broke some downstream packages due to picking a newer version of `recursion-schemes`. I think it's just a matter of fixing up the import lists, do you want to do that or go back to fixing `ral-0.1`